src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  background-color: #e5e5e5;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+
+  --secondary: #616070;
+  --main-accent: #060b35;
+  --white: #fff;
+  --blue-accent: #00acdc;
+}
+
+body,
+div,
+a,
+h1,
+h2,
+img,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
Create a separate file for card block
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/e610da9c-3290-4188-ae1e-278b0291f6df)